src/vm/sampleprofiler.cpp

```diff
@@ -195,4 +221,30 @@ void SampleProfiler::WalkManagedThreads()
     }
 }
 
-#endif // FEATURE_PERFTRACING
+void SampleProfiler::PlatformSleep(long nanoseconds){
```

Comment:
A few standard things for these functions.  They should follow the coding guidelines and have { and } on separate lines.  Also each new function needs a contract.